internal/cache/lru/lru.go

```diff
@@ -267,3 +268,17 @@ func (c *Cache) UpdateWithoutChangingOrder(
 
 	return nil
 }
+
+func (c *Cache) EraseEntriesWithGivenPrefix(
+	prefix string) (value []ValueType) {
```

Comment:
Please dont add func arguments in new line. 

Already commented in earlier PR
https://github.com/GoogleCloudPlatform/gcsfuse/pull/2045